static/app/views/dashboardsV2/widgetBuilder/buildSteps/visualizationStep.tsx

```diff
@@ -106,15 +107,19 @@ export function VisualizationStep({
 
 const VisualizationWrapper = styled('div')<{displayType: DisplayType}>`
   padding-right: ${space(2)};
+  ${WidgetCardPanel} {
+    height: initial;
+  }
   ${p =>
     p.displayType === DisplayType.TABLE &&
     css`
       ${TableCell} {
         /* 24px ActorContainer height + 16px top and bottom padding + 1px border = 41px */
         height: 41px;
       }
-      /* total size of a table, if it would display 5 rows of content */
-      height: 300px;
-      overflow: hidden;
+      ${WidgetCardPanel} {
+        /* total size of a table, if it would display 5 rows of content */
+        height: 301px;
```

Comment:
do we need to add one more pixel?

Reply:
![image](https://user-images.githubusercontent.com/29228205/165078594-03a5d108-2329-4289-af9e-d3d3c57536fc.png)

Reply:
I believe so, right now in production I can see tiny scrollbars in the table if you try to scroll inside it. Can you?

Reply:
oh yes... it is not very visible... good catch!